calculator.go

```diff
@@ -1,13 +1,89 @@
 // Package calculator provides a library for simple calculations in Go.
 package calculator
 
+import (
+	"errors"
+	"fmt"
+	"math"
+)
+
 // Add takes two numbers and returns the result of adding them together.
-func Add(a, b float64) float64 {
-return a + b
+func Add(nums ...float64) float64 {
+	var result float64 = nums[0]
+
+	for _, n := range nums[1:] {
+		result = result + n
+	}
+	return result
 }
 
 // Subtract takes two numbers and returns the result of subtracting the second
 // from the first.
-func Subtract(a, b float64) float64 {
-	return b - a
+func Subtract(nums ...float64) float64 {
+	var result float64 = nums[0]
+
+	for _, n := range nums[1:] {
+		result = result - n
```

Comment:
There's a shorthand form for this: `result -= n`.

Reply:
addressed in 6962c56